tests/integration/workflows/java_gradle/test_java_gradle.py

```diff
@@ -0,0 +1,138 @@
+import os
+import shutil
+import tempfile
+
+from zipfile import ZipFile
+from unittest import TestCase
+
+from aws_lambda_builders.builder import LambdaBuilder
+from aws_lambda_builders.exceptions import WorkflowFailedError
+
+
+class TestJavaGradle(TestCase):
```

Comment:
Love the extensive coverage of integration tests! Thanks a lot for taking the time. Really helps pushing the boundary for code quality